public/controllers/login-signup.js

```diff
@@ -85,6 +86,31 @@ controller.controller('Login', ['ConfigService', '$scope', '$http', '$location',
                 alert("login failed, reason: " + err.reason);
             });
         };
+        $scope.resetPassword = function(){
+            var url = ConfigService.serverIp + '/forgetPassword';
+            $http.post(url, {username: $scope.forgottenUsername}).success(function(response) {
+                alert(response);
```

Comment:
success response says: "Queued. Thanks you." Shouldn't it say something like "An email has been sent to ****ali@gmail.com, please check that to reset your password." ?